src/features/search/screens/SearchRepositoryScreen.js

```diff
@@ -1,16 +1,132 @@
 // @flow
 
 import React, {Component} from 'react';
-import {View, Text} from 'react-native';
+import {
+  View,
+  Text,
+  ScrollView,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+import {Ionicons, Octicons, FontAwesome} from '@expo/vector-icons';
 
-class SearchRepositoryScreen extends Component<{}> {
+import type {NavigationScreenProp} from 'react-navigation';
+
+type Repo = {
+  fullName: string,
+  description: string,
+  starsCount: number,
+  forksCount: number,
+  language: string,
+  fork: boolean,
+};
+
+type Props = {
+  navigation: NavigationScreenProp<[]>,
```

Comment:
Yes, this could be anything but some state requires named props. So, object is good for this.